### ISSUE TITLE
[dataclasses] Update SPE Templates (Trac #2174)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2166]
The branches are located here:
http://code.icecube.wisc.edu/svn/projects/DOMLauncher/branches/dual_exp_spe
http://code.icecube.wisc.edu/svn/projects/dataclasses/branches/dual_exp_spe

Merge these into the respective trunks and include in the next release.

Migrated from https://code.icecube.wisc.edu/ticket/2174
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:18", 
    "description": "The branches are located here:\nhttp://code.icecube.wisc.edu/svn/projects/DOMLauncher/branches/dual_exp_spe\nhttp://code.icecube.wisc.edu/svn/projects/dataclasses/branches/dual_exp_spe\n\nMerge these into the respective trunks and include in the next release.", 
    "reporter": "olivas", 
    "cc": "saxani, olivas", 
    "resolution": "fixed", 
    "_ts": "1550067318169976", 
    "component": "combo core", 
    "summary": "[dataclasses] Update SPE Templates", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2018-07-17T15:13:40", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180717T15:13:49: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20180730T16:27:29: 
* olivas changed cc from "" to "saxani, jbraun"
* olivas commented:

> Apparently this is going to be a little more complicated than a simple svn merge.  Going to try to hash this out with Spencer and Jim this week.
>

**Comment by icecube-trac** [comment]
Trac update at 20180922T10:04:16: olivas changed milestone from "Summer Solstice 2018" to "Autumnal Equinox 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181001T21:57:36: olivas changed priority from "normal" to "blocker"

**Comment by icecube-trac** [comment]
Trac update at 20181029T13:30:02: 
* olivas commented:

> Added a module to inject the new SPE parameters into a C frame from JSON file.  Ideally these would live in the DB.  Transferring this ticket to Jim, since now all we need is a GCD file, and those constants loaded into the DB.


* olivas changed cc from "saxani, jbraun" to "saxani, olivas"
* olivas changed status from "accepted" to "assigned"
* olivas changed owner from "olivas" to "jbraun"

**Comment by icecube-trac** [comment]
Trac update at 20181029T15:29:33: jbraun commented:

> Because we chose to overwrite the mean ATWD/FADC charge values in dataclasses with Spencer’s data, we can’t put this data into the database until the 2019 run start.  Otherwise, we’d end up using Spencer’s calibration constants for processing the current data at SPS.  We'll have to patch simulation GCD files until then.

**Comment by icecube-trac** [comment]
Trac update at 20181029T15:30:26: jbraun commented:

> Keep this ticket open until we insert Spencer's data into the database in 2019.

**Comment by icecube-trac** [comment]
Trac update at 20181119T18:06:23: 
* jbraun commented:

> In changeset 166943:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="166943"
> Support Spencer's SPE calibration JSON files.  Closes #2166.
> ```


* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T04:02:40: nwhitehorn commented:

> In changeset 170604:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170604"
> Support Spencer's SPE calibration JSON files.  Closes #2166.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:18: 
* olivas commented:

> Milestone Autumnal Equinox 2018 deleted


* olivas changed milestone from "Autumnal Equinox 2018" to ""
